app/src/main/java/com/example/nigeriatelemedicineapp/utils/DateUtlis.kt

```diff
@@ -0,0 +1,59 @@
+package com.example.nigeriatelemedicineapp.utils
+
+class DateUtlis {
+
+    companion object {
+
+        fun checkDateFormat(date: String) : Boolean {
+            return DateUtlis().validate(date)
+        }
+
+        fun getCorrectDateFormat(date:String) : String{
+           val StringArray = date.split("-")
+            var correctFormat: String=""
+            correctFormat= correctFormat+StringArray[2]+"-"+StringArray[1]+"-"+StringArray[0]
```

Comment:
Do we know for sure that this won't throw an outofbounds excpetion? Maybe add a check to make sure the result of `date.split("-")` is 3 elements long.

Reply:
Yes actually this function is only called once the date is correctly verified.